pywbem/_server.py

```diff
@@ -762,6 +764,34 @@ def get_central_instances(self, profile_path, central_class=None,
             implementation supports only the scoping class methodology.
             `None` will cause the scoping class methodology not to be attempted.
 
+          reference_direction (:term:`string`):
+            Defines the usage of the ResultRole parameter in the association
+            request that determines the relation between component registered
+            profiles.  The possible values are 'dmtf' and 'snia' the default is
+            'dmtf'. This parameter exists because of issues between DMTF and
+            SNIA in the the definition of CIM_ReferencedProfile and its
+            subclasses.
+
+            Generally the DMTF definition documented in DSP1033 and the
+            DMTF PRP (DSP1001) is that:
+
+            * Antecedent = referenced profile = component profile
+            * Dependent = referencing profile = autonomous profile
+
+            At the same time, for historical reasons, the SNIA SMI specification
+            is ambiguous in that the definition varies by document.
+
+            SMI-S PRP in book xxx specification is that:
```

Comment:
Don't abbreviate PRP, but use its proper profile name "Profile Registration Profile".
It is in the "Common Profiles" book.

Reply:
DONE